.github/workflows/go-build-test.yml

```
@@ -72,7 +71,7 @@ jobs:
           go mod edit -replace=github.com/nextmv-io/sdk=${{env.SDK_PATH}}
 
           # Build the plugins.
-          REMOVE_TRIMPATH=1 NEXTMV_LIBRARY_PATH=${{ env.NEXTMV_LIBRARY_PATH }} bash scripts/build.sh
+          REMOVE_TRIMPATH=1 NEXTMV_LIBRARY_PATH=${{ env.NEXTMV_LIBRARY_PATH }} bash scripts/build.sh > /dev/null 2>&1
```

```suggestion
          REMOVE_TRIMPATH=1 NEXTMV_LIBRARY_PATH=${{ env.NEXTMV_LIBRARY_PATH }} bash scripts/build.sh &> /dev/null
```

This is equivalent to this, correct? I am used to doing this, but I think I get how yours works. :thinking: 
Anyway, just curious not at all blocking.
Also, I get it right that this means to silence stdout & stderr, right?

---

Yeah the idea is to silence all output.